module/condition/migrations/Version20201119103017.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Migration;
+
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20201119103017 extends AbstractErgonodeMigration
+{
+
+    public function up(Schema $schema): void
+    {
+        $languages = $this->getDictionaryActive();
+
+        $this->addSql(sprintf(
+            'WITH conditions_list as (
+                SELECT id, (\'{\'|| arr.position -1 || \',language}\')::text[] as path
+                FROM condition_set,
+                    jsonb_array_elements(conditions) with ordinality arr(item_object, position)
+                where arr.item_object::jsonb ->> \'type\' = \'PRODUCT_HAS_STATUS_CONDITION\')
+            UPDATE condition_set
+            set conditions = jsonb_set(conditions, conditions_list.path, \'%s\')
```

Comment:
Please write language constructs (like `set`)  with the upper case in the entire query and not just parts of it. Also reformat it - i.e. in this line indentation is missing